validator-api/src/cache/mod.rs

```diff
@@ -37,6 +37,8 @@ struct ValidatorCacheInner {
 
     current_mixnode_active_set_size: AtomicUsize,
     current_gateway_active_set_size: AtomicUsize,
+
+    total_mix_stake: RwLock<Cache<u128>>,
```

Comment:
Does it really make sense to wrap it in cache? Also is there no AtomicU128 to get rid of the lock?

Reply:
Unfortunately no, AtomicU128 is in a [weird limbo state](https://github.com/rust-lang/rust/issues/32976#issuecomment-893117019)...